dlp/spec/data/test.txt

```diff
@@ -1 +1 @@
-Robert Frost was an American poet.
\ No newline at end of file
+My phone number is (555) 456-7890 and my email address is gary@somedomain.com.
```

Comment:
`example.com` <-- this is basically the `(555)` for the Web

https://en.wikipedia.org/wiki/Example.com